mslib/msui/msui.py

```diff
@@ -591,17 +591,19 @@ def remove_plugins(self):
         self.export_plugins = {}
 
     def handle_import_local(self, extension, function, pickertype):
-        filename = get_open_filename(
+        filename = get_open_filenames(
             self, "Import Flight Track",
             self.last_save_directory,
             f"Flight Track (*.{extension});;All files (*.*)",
             pickertype=pickertype)
         if self.local_active:
             if filename is not None:
-                self.last_save_directory = fs.path.dirname(filename)
-                self.create_new_flight_track(filename=filename, function=function)
+                for name in filename:
```

Comment:
maybe filenames instead of filename?

Reply:
true, better wording. it can still be only one file.

Reply:
Fine either way. I tend to use plural for lists or tuples independent of length (can be empty as well).